@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -179,20 +177,24 @@ export class OmvDecodedTileEmitter implements IOmvEmitter {
      * Creates the Point of Interest geometries for the given feature.
      *
      * @param layer Tile's layer to be processed.
+     * @param extents Tile's layer extents.
      * @param feature The current feature containing the main geometry.
```

Comment:
rename to "geometry", or rename the actual parameter to "feature"

Reply:
Good catch!